TASK-5/TASK-5/Menu.cs

```diff
@@ -0,0 +1,176 @@
+using Task_5.Generators;
+using Task_5.Models;
+
+namespace Task_5;
+
+public class Menu
+{
+    private static readonly Random _amount = new();
+    
+    public static void MainMenu(List<User> users)
+    {
+        var exit = true;
+        while (exit)
+        {
+            Console.WriteLine();
+            Console.WriteLine("You are in Main Menu, you can do: \n" +
+                              "1. Display every user.\n" +
+                              "2. Display user's goods with total cost.\n" +
+                              "3. Add new user from the console.\n" +
+                              "4. Add goods to user's bag.\n" +
+                              "5. Remove goods from user's bag.\n" +
+                              "To exit, press any number, except number above (0, for example)");
+            string input;
+            int menuItem;
+            do
+            {
+                Console.Write("Please, enter option number: ");
+                input = Console.ReadLine();
+            } while (!int.TryParse(input, out menuItem));
+
+            switch (menuItem)
+            {
+                case 1:
+                    DisplayEveryClient(users);
+                    break;
+                case 2:
+                    DisplayClientPurchases(users);
+                    break;
+                case 3:
+                    GenerateUserFromConsole(users);
+                    break;
+                case 4:
+                    AddItemToBag(users);
+                    break;
+                case 5:
+                    RemoveItemFromBag(users);
+                    break;
+                default:
+                    exit = false;
+                    break;
+            }
+        }
+    }
+
+    public static List<User> GenerateFiveUsers()
```

Comment:
Make it GenerateUsers and pass count parameter.

Reply:
UserGenerator.GenerateUserWithItems(int count) -> userWithItems.Generate(int count)